modules/customizing-the-login-page.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * web_console/customizing-the-web-console.adoc
+
+[id="customizing-the-login-page_{context}"]
+= Customizing the login page
+
+Create Terms of Service information with custom login pages. Custom login pages
+can also be helpful if you use a third-party login provider, like GitHub or
```

Comment:
s/like/such as/
**like** _conj_
Do not use to mean “such as.”

Reply:
@bobfuru Yes, good catch. Thank you!